src/deploy-cromwell-on-azure/scripts/mount.blobfuse

```diff
@@ -14,23 +20,22 @@ function write_log() {
 
 account_name=$(echo $3 | grep -Po '(?i)account_name=\K([^,]*)')
 container_name=$(echo $3 | grep -Po '(?i)container_name=\K([^,]*)')
-account_key=$(echo $3 | grep -Po '(?i)account_key=\K([^,]*)')
-sas_token=$(echo $3 | grep -Po '(?i)sas_token=\K([^,]*)')
+account_key=$(echo $3 | grep -Po '(?i)account_key=\K([^,]*)' || :)
```

Comment:
: means "noop" 
|| : means: if the preceding command fails, run noop, in other words ignore the error and continue